### ISSUE TITLE
codespell

### ISSUE THREAD

**Issue by dagewa** [dials/dials#2529]
I ran [codespell](https://github.com/codespell-project/codespell) on `dials/`. Here is a filtered list of the output (after removing lots of lines referring to spellings in the sphinx documentation):

```
./CHANGELOG.rst:184: minimial ==> minimal
./CHANGELOG.rst:187: succesfully ==> successfully
./tests/util/test_observer.py:90: warpping ==> wrapping, warping
./tests/command_line/test_import.py:113: ore ==> or
./tests/command_line/test_import.py:770: completess ==> completes, completeness
./tests/command_line/test_search_beam_position.py:93: perturbate ==> perturb
./tests/command_line/test_search_beam_position.py:102: perturbate ==> perturb
./tests/command_line/test_export_mosflm.py:15: fo ==> of, for, to, do, go
./tests/command_line/test_export_mosflm.py:16: fo ==> of, for, to, do, go
./tests/algorithms/refinement/test_stills_spherical_relp_derivatives.py:61: Simplied ==> Simplified, Implied
./tests/algorithms/polygon/clip/test_clipping.py:231: inbetween ==> between, in between
./tests/algorithms/polygon/clip/test_clipping.py:245: inbetween ==> between, in between
./tests/algorithms/polygon/clip/test_clipping.py:253: inbetween ==> between, in between
./tests/algorithms/polygon/clip/test_clipping.py:261: inbetween ==> between, in between
./tests/algorithms/polygon/clip/test_clipping.py:269: inbetween ==> between, in between
./tests/array_family/test_identifiers_handling.py:11: idenfitiers ==> identifiers
./src/dials/util/image_grouping.py:903: tempaltes ==> templates
./src/dials/util/ioutil.py:21: tyoe ==> type, toey, toe
./src/dials/util/options.py:462: interpretor ==> interpreter
./src/dials/util/options.py:486: interpretor ==> interpreter
./src/dials/util/image_viewer/slip_viewer/tile_generation.py:270: coordintes ==> coordinates
./src/dials/command_line/stills_process.py:280: distince ==> distance
./src/dials/command_line/show.py:318: reinstatiate ==> reinstantiate
./src/dials/command_line/integrate.py:9: integate ==> integrate
./src/dials/command_line/reindex.py:42: integated ==> integrated
./src/dials/command_line/symmetry.py:107: Signficance ==> Significance
./src/dials/command_line/ssx_index.py:170: sucessfully ==> successfully
./src/dials/command_line/plot_Fo_vs_Fc.py:3: Fo ==> Of, For, To, Do, Go
./src/dials/command_line/plot_Fo_vs_Fc.py:104: Fo ==> Of, For, To, Do, Go
./src/dials/command_line/plot_Fo_vs_Fc.py:124: Fo ==> Of, For, To, Do, Go
./src/dials/command_line/plot_Fo_vs_Fc.py:153: Fo ==> Of, For, To, Do, Go
./src/dials/command_line/plot_Fo_vs_Fc.py:159: Fo ==> Of, For, To, Do, Go
./src/dials/command_line/plot_Fo_vs_Fc.py:226: Fo ==> Of, For, To, Do, Go
./src/dials/command_line/ssx_integrate.py:419: wont ==> won't
./src/dials/algorithms/indexing/ssx/processing.py:104: Reenable ==> Re-enable
./src/dials/algorithms/spatial_indexing/detect_collisions.h:413: wholely ==> wholly
./src/dials/algorithms/refinement/refinement_helpers.py:33: nd ==> and, 2nd
./src/dials/algorithms/refinement/sparse_engine.py:102: numberically ==> numerically
./src/dials/algorithms/refinement/restraints/restraints_helpers.h:36: daa ==> data
./src/dials/algorithms/refinement/restraints/restraints.py:154: daa ==> data
./src/dials/algorithms/refinement/restraints/restraints.py:158: daa ==> data
./src/dials/algorithms/image/connected_components/connected_components.h:317: dimensins ==> dimensions
./src/dials/algorithms/profile_model/ellipsoid/algorithm.py:289: parmameters ==> parameters
./src/dials/algorithms/profile_model/ellipsoid/algorithm.py:328: parmameters ==> parameters
./src/dials/algorithms/statistics/kolmogorov_smirnov_one_sided_distribution.h:77: fo ==> of, for, to, do, go
./src/dials/algorithms/statistics/kolmogorov_smirnov_one_sided_distribution.h:157: fo ==> of, for, to, do, go
./src/dials/algorithms/scaling/scaling_utilities.py:196: probablity ==> probability
./src/dials/algorithms/scaling/scaling_library.py:603: compability ==> compatibility
./src/dials/algorithms/scaling/parameter_handler.py:17: mimimisation ==> minimisation
./src/dials/algorithms/scaling/reflection_selection.py:280: defecit ==> deficit
./src/dials/algorithms/scaling/reflection_selection.py:297: defecit ==> deficit
./src/dials/algorithms/scaling/reflection_selection.py:302: defecit ==> deficit
./src/dials/algorithms/scaling/reflection_selection.py:303: defecit ==> deficit
./src/dials/algorithms/scaling/reflection_selection.py:309: defecit ==> deficit
./src/dials/algorithms/scaling/reflection_selection.py:314: defecit ==> deficit
./src/dials/algorithms/scaling/reflection_selection.py:318: defecit ==> deficit
./src/dials/algorithms/scaling/reflection_selection.py:326: defecit ==> deficit
./src/dials/algorithms/scaling/reflection_selection.py:328: defecit ==> deficit
./src/dials/algorithms/scaling/cross_validation/crossvalidator.py:47: discreet ==> discrete
./src/dials/algorithms/scaling/cross_validation/cross_validate.py:57: discreet ==> discrete
./src/dials/algorithms/integration/kapton_2019_correction.py:250: Insuffient ==> Insufficient
./src/dials/algorithms/integration/kapton_2019_correction.py:253: permuations ==> permutations
./src/dials/algorithms/integration/kapton_2019_correction.py:267: explictly ==> explicitly
./src/dials/algorithms/integration/kapton_2019_correction.py:324: descibe ==> describe
./src/dials/algorithms/integration/algorithms.h:87: relfection ==> reflection
./src/dials/algorithms/integration/algorithms.h:119: relfection ==> reflection
./src/dials/algorithms/integration/fit/fitting.h:33: dimensons ==> dimensions
./src/dials/report/analysis.py:428: colum ==> column
./src/dials/array_family/reflection_table.h:95: Integated ==> Integrated
./src/dials/array_family/boost_python/reflection_table_suite.h:180: identifers ==> identifiers
./cmake/Modules/AlwaysColourCompilation.cmake:4: Unfortuantely ==> Unfortunately
./cmake/Modules/FindCCTBX.cmake:117: modile ==> module, mobile
./cmake/Modules/FindCCTBX.cmake:191: speciic ==> specific
./newsfragments/2511.feature:1: sitation ==> situation
```
I'm happy to work through these when I get a spare hour or so.

**Comment by ndevenish** [comment]
It's amazing how they snuck in since the last codespell https://github.com/dials/dials/commit/ea203d3a1e522fc2590176f0061c6378218334f8 🙂

**Comment by dagewa** [comment]
Ah your codespell spell in the link looks useful. I think the above was just a plain reporting run

**Comment by ndevenish** [comment]
I couldn't find a way to eliminate short corrections 😔

**Comment by dagewa** [comment]
Most of these fixed by #2550
